### PR TITLE
KOGITO-4539: Tests of large models

### PR DESCRIPTION
**Pull request by jomarko**
This PR adds tests of large models. Also updates pom.xml to ensure they are not started by default.

**Dependent on**:
* #3578 - [update] - merged

**Ensemble together with**:
* https://github.com/kiegroup/droolsjbpm-build-bootstrap/pull/1600

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

* Retest PR: <b>jenkins retest this</b>
* A full downstream build: <b>jenkins do fdb</b>
* A compile downstream build: <b>jenkins do cdb</b>
* A full production downstream build: <b>jenkins do product fdb</b>
* An upstream build: <b>jenkins do upstream</b>
</details>

### CODE REVIEW

**Review by jstastny-cz — APPROVED** [review]

**Comment by karreiro** [comment]
@jomarko Could you please solve conflicts? (I'd like to run the new switch locally - btw, great job creating a new specific switch for large-models-tests - thanks!)

**Comment by jomarko** [comment]
@karreiro conflicts resolved

**Comment by jomarko** [comment]
@jstastny-cz sorry to asking a re-review, the PR (currently ensemble of two) has changed slightly since your review.

File: kie-wb-common-dmn/kie-wb-common-dmn-webapp-kogito-runtime/pom.xml [hunk 1]
```diff
@@ -33,7 +33,9 @@
     <skipITs>false</skipITs>
     <org.kie.dmn.kogito.browser.headless>true</org.kie.dmn.kogito.browser.headless>
     <!-- We do not want to run selenium tests in default profile -->
-    <kogito.it.tests.excludes>**/*IT.java</kogito.it.tests.excludes>
+    <kogito.large.model.it.tests>**/large/model/*IT.java</kogito.large.model.it.tests>
```
Comment (jstastny-cz) [inline kie-wb-common-dmn/kie-wb-common-dmn-webapp-kogito-runtime/pom.xml:36]: the comment above is not fulfilled, we're including `kogito.selenium.it.tests` in default profile now, isn't it?

Comment (jomarko) [reply kie-wb-common-dmn/kie-wb-common-dmn-webapp-kogito-runtime/pom.xml:35]: well spotted, thank you, resolved, now checking why upgrade of webdrivermanager causes build to fail, as locally worked.

**Comment by jomarko** [comment]
Jenkins please retest this

**Review by karreiro — APPROVED** [review]
Great stuff! Thank you, @jomarko :-)

**Comment by jomarko** [comment]
Jenkins please retest this

**Comment by jomarko** [comment]
Jenkins please retest this

**Review by jstastny-cz — APPROVED** [review]
LGTM, thanks.

**Comment by jomarko** [comment]
@kiegroup/gatekeepers please merge together with https://github.com/kiegroup/droolsjbpm-build-bootstrap/pull/1600

**Pull request merged**